### DISCUSSION TITLE
ux support for receiving test results in realtime from a local testrun and/or triggering new runs

### DISCUSSION THREAD

**Discussion by RonnyPfannschmidt** [pytest-dev/pytest-html#541] [Ideas]
in response to https://github.com/pytest-dev/pytest-html/pull/479#issuecomment-1200867465

i'd like to be able to "live-stream testr run results into a browser vi as i run them locally, optionally triggering pytest reruns on changes

this would essentially be something like looponfail, just in the browser

this would require a local webserver to serve the assets + event stream websockets with test results

this would be a consideration for strictly after the new fe/be have landed with feature parity

**Comment by BeyondEvil** [comment]
Would you mind elaborating on the second part? 🤔 

> optionally triggering pytest reruns on changes
